buildSrc/src/main/kotlin/otel.android-app-conventions.gradle.kts

```
@@ -0,0 +1,29 @@
+import com.android.build.api.dsl.LibraryExtension
+import org.gradle.api.publish.maven.MavenPublication
+
+plugins {
+    id("com.android.application")
+    id("otel.errorprone-conventions")
+}
+
+android {
+    namespace = "io.opentelemetry.android"
+    compileSdk = 33
```

The parent is now 34....we should try and be consistent right?

```suggestion
    compileSdk = 34
```

---

I've applied the changes